staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -1288,6 +1288,13 @@ func (r Result) StatusCode(statusCode *int) Result {
 	return r
 }
 
+// ContentType returns the "Content-Type" response header into the passed
+// string, returning the Result for possible chaining.
```

Comment:
does this need a similar disclaimer as StatusCode that it is only valid if no error was returned?

Reply:
Good catch. Yes--similar disclaimer needed and added.